osu.Game/Overlays/Mods/ModSelectOverlay.cs

```diff
@@ -64,6 +64,26 @@ public abstract partial class ModSelectOverlay : ShearedOverlayContainer, ISampl
             }
         }
 
+        /// <summary>
+        /// Search term applied on mod overlay
+        /// </summary>
```

Comment:
This xmldoc is redundant to the point of bringing no meaning. It might as well not be there.